### PR TITLE
Opsætning af OS2DisplayPC

### PR DESCRIPTION
**Pull request by agger-magenta**
Dette løser opgave #33102 i Redmine.

Det er testet, at dette kan udføres på en ny Ubuntu Server-installation med custom image fra løsningen på #33103: 

>En Ubuntu server bliver installeret, scriptet bliver kørt og herefter tjekkes at computeren nu er tilmeldt i administrationssystemet.

### CODE REVIEW

**Comment by dals83** [comment]
An idea could be to divide the scripts needed to setup an os2displaypc and an os2borgerpc into different folders. I had some difficulties figuring out what was needed for what an when some scripts was shared.

File: image/scripts/install_dependencies.sh [hunk 1]
```diff
@@ -72,6 +67,23 @@ if [ "$PKGSTOINSTALL" != "" ]; then
     apt-get -y clean
 fi
 
+
+# Things to get rid of. Factor out to file if many turn up.
+grep -w "ii  deja-dup" /tmp/installed-package-list.txt > /dev/null
```
Comment (dals83) [inline image/scripts/install_dependencies.sh]: This script should actually be tested with an os2borgerpc installation while it is now used for both types of pc installations.

Comment (agger-magenta) [reply image/scripts/install_dependencies.sh]: True!

**Comment by agger-magenta** [comment]
> An idea could be to divide the scripts needed to setup an os2displaypc and an os2borgerpc into different folders. I had some difficulties figuring out what was needed for what an when some scripts was shared.

Yes. Only I found it very tempting to share the `install_dependencies` script. :) 

But basically all the `display_setup` script is doing is install the dependencies needed to connect to the admin system, so it's more of a "borgerPC Server" script than actually display-related.

**Comment by dals83** [comment]
> > An idea could be to divide the scripts needed to setup an os2displaypc and an os2borgerpc into different folders. I had some difficulties figuring out what was needed for what an when some scripts was shared.
> 
> Yes. Only I found it very tempting to share the `install_dependencies` script. :)
> 
> But basically all the `display_setup` script is doing is install the dependencies needed to connect to the admin system, so it's more of a "borgerPC Server" script than actually display-related.

Yes maybe the name we have been given the project (OS2displayPC) is misleading. It could instead be called OS2borgerPC-server or something. I still think dividing the borgerPC scripts and displayPC scripts into different folders is a good idea. Then all the shared scripts can stay where they are...

**Pull request merged**